trusted_application_demo/ca_security_api/src/teec_utils.c

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright (c) 2018, ARM Limited.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <tee_client_api.h>
+
+#include <ca_security_api/teec_utils.h>
+
+void
+ca_teec_check_result(TEEC_Result res, const char * function, const char * errmsg)
```

Comment:
As this doesn't return anything but just print to console, I would suggest using a function name that reflect that